### PR TITLE
Promise nwafor

### PR DESCRIPTION
**Pull request by promiseNwafor**
- Added regular textfield with icon option on th right
- Added color files

### CODE REVIEW

File: src/utils/Colors.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const Colors = {
+    PRIMARY = "#850DCE",
+    SECONDARY = "#FAF2FF",
+    SECONDARY_LIGHT = "#E8C4FF",
+    MAIN_ACCENT = "#F6B05D",
+    SUCCESS = "#22B02E",
+    ERROR = "#D82C0D",
+    LOADING_SEA = "#01369A",
+    GREY = "#474747",
+    INFORMATION = "#0F62FE",
```
Comment (chuksdozie) [inline src/utils/Colors.js:10]: be more descriptive

File: src/utils/Colors.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const Colors = {
```
Comment (chuksdozie) [inline src/utils/Colors.js:1]: use all caps for the object name ie. COLORS

File: src/utils/Colors.js (hunk 1 above)
Comment (chuksdozie) [reply src/utils/Colors.js:10]: What information?

**Review by chuksdozie — CHANGES_REQUESTED** [review]
nothing much, just make these corrections

**Review by chuksdozie — COMMENTED** [review]
send a screenshot of the component after it is working

File: src/utils/Colors.js (hunk 2 above)
Comment (promiseNwafor) [reply src/utils/Colors.js:1]: Noted

File: src/utils/Colors.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    PRIMARY = "#850DCE",
+    SECONDARY = "#FAF2FF",
+    SECONDARY_LIGHT = "#E8C4FF",
+    MAIN_ACCENT = "#F6B05D",
+    SUCCESS = "#22B02E",
+    ERROR = "#D82C0D",
+    LOADING_SEA = "#01369A",
```
[the remark this answers is not in the document]
Comment (promiseNwafor) [reply src/utils/Colors.js:8]: I was trying to use the labels the designers used. Is it not necessary?

**Pull request closed without merging**